src/EditorFeatures/Core/Implementation/CommentSelection/CommentTrackingSpan.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection
+{
+    /// <summary>
+    /// Wrapper around a TextSpan that holds extra data used to create a tracking span.
+    /// </summary>
+    internal struct CommentTrackingSpan
+    {
+        public TextSpan TrackingTextSpan { get; }
+
+        // In some cases, the tracking span needs to be adjusted by a specific amount after the changes have been applied.
+        // These fields store the amount to adjust the span by after edits have been applied.
+        public int AmountToAddToStart { get; }
+        public int AmountToAddToEnd { get; }
```

Comment:
readonly fields, since you don't actually do any comptation for them?

Reply:
could you just store two spans here and explain which purpose each one serves?  it's not clear to me.  htanks!

Reply:
See comment below on the same - expanded comment as well.
{get;} is readonly already right?